uint8 val

uint8 SUCCESS=1
uint8 FAILURE=0
uint8 NONE = 100

uint8 PLANNING_FAILED=1
uint8 CONTROL_FAILED=2
uint8 NO_IK_SOLUTION=3
uint8 INVALID_FRAME=4
uint8 NO_GOAL_POSE=5
uint8 UNKNOWN_COMMAND=6
